src/js/module/K2previousissues/K2previousissues.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import PreviousIssuesButtons from './PreviousIssuesButton';
+
+export default function () {
+    return {
+        draw() {
+            const root = createRoot(document.getElementsByClassName('k2previousissue-wrapper')[0]);
```

Comment:
NAB - I'm noticing that the files are usually named "previous issue**s**" (with an S) while the css is missing the `s` - could be nice to make that consistent either way

```suggestion
            const root = createRoot(document.getElementsByClassName('k2previousissues-wrapper')[0]);
```